Live/LiveEventWithDVR/README.md

```diff
@@ -1,29 +1,31 @@
-# Live event with DVR
+# Live Event with DVR
 
-This sample demonstrates how to create and use LiveEvents and LiveOutputs in the v3 Media Services API. It performs the following tasks:
+This sample demonstrates how to create and use Live Events and Live Outputs in the v3 Media Services API. It performs the following tasks:
 
 1. Creates a "standard" pass-through LiveEvent.
 1. Starts monitoring the LiveEvent using Event Grid and Event Hub.
 1. Creates an Asset and an AssetFilter with 5 minutes sliding window and 30 seconds seek back time.
-1. Creates a LiveOutput with 25 hours time span.
-1. Creates a StreamingLocator and associate the AssetFilter with it.
-1. Prints urls for the LiveEvent.
-1. Prints playback urls for the event archive after the LiveEvent stops.
+1. Creates a Live Output with 25 hours time span.
```

Comment:
Let's make this window smaller. How about 6 hours?

Reply:
Changed to 1 hour to match the code.